geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -1,5 +1,19 @@
 This file includes migration steps for each release of c2cgeoportal.
 
+
+Version 2.4.0
+=============
+
+Information
+-----------
+
+1. The `docker-service` section in the vars files is removed, and the `docker-compose.yaml.mako`.
+   Now we will have a `docker-compose-lib.yaml` file with a base of the services,
```

Comment:
`Now we have` instead of `Now we will have` ?

Reply:
+1